### DISCUSSION TITLE
timeline for 3.3 release

### DISCUSSION THREAD

**Discussion by blacknell** [ESPresense/ESPresense#986] [General]
@DTTerastar any thoughts on timeline for release? i'd really like to take advantage of DS18B20 sensor support

**Comment by DTTerastar** [comment]
I think i'm going to go all the way to 4.0 with all the mqtt auto discovery fixed for Home Assistant.

**Reply by blacknell** [reply]
Do you have a guess as the timeline?

**Reply by DTTerastar** [reply]
I've fixed some memory issues that were really bad.  Getting closer to a release.

**Reply by DTTerastar** [reply]
Released v3.3 just now!
